matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/SyncService.kt

```diff
@@ -62,7 +62,7 @@ interface SyncService {
     /**
      * Get the [SyncRequestState] as a LiveData.
      */
-    fun getSyncRequestStateLive(): LiveData<SyncRequestState>
+    fun getSyncRequestStateFlow(): SharedFlow<SyncRequestState>
```

Comment:
Can I ask you to update the comment above please? (just replace `LiveData` by `SharedFlow`).